docker-app/qfieldcloud/core/views/package_views.py

```diff
@@ -85,13 +85,16 @@ class LegacyLatestPackageView(views.APIView):
     def get(self, request, project_id):
         """Get last project package status and file list."""
         project = Project.objects.get(id=project_id)
+        last_package_job = project.latest_package_job_for_user(request.user)
```

Comment:
```suggestion
        latest_package_job = project.latest_package_job_for_user(request.user)
```

Just to make sure we use the same name everywhere. I think it is worth it to grep for `last.*package` in the code, there might be a few other places where we are left with the old name.